apps/system/js/homescreen_window.js

```diff
@@ -89,12 +89,12 @@
    */
   HomescreenWindow.prototype.setBrowserConfig =
     function hw_setBrowserConfig(_manifestURL) {
-      // FIXME: don't hardcode that.
-      var manifestURL = 'chrome://gaia/content/homescreen/manifest.webapp';
-      this.url = 'chrome://gaia/content/homescreen/index.html#root';
-      var url = new URL(manifestURL);
-      this.origin = url.origin;
-      this.manifestURL = manifestURL;
+      // FIXME: Why is not manifest.webapp stored?
```

Comment:
Restored to what we have in master (the only difference is adding /manifest.webapp)